src/tracks/Pattern.cpp

```diff
@@ -125,6 +125,55 @@ void Pattern::init()
 
 	changeLength( length() );
 	restoreJournallingState();
```

Comment:
`init()` is also run for the copy constructor, which is correctly initialized. This code should only run for the other constructor and before `ensureBeatNotes()`, thus before `init()`.